### DISCUSSION TITLE
Problem with Webradio Album Name And Cover Art

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#557] [Fixed Issues]
This URL, for instance, http://streams.radiobob.de/bob-rockabilly/mp3-192 used to display album name and cover art on the playback page with no problem. Now there is the default image and no album name until I go to the Playlist page and return to the playback page. When the track changes, the default image is displayed again. Occasionally the stream is terminated when the cover art appears.

Radio Paradise streams have cover art as normal.

rAudio 1 • 20211011
5.10.63-18-raspberrypi4-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2

EDIT: Just to add that I tried a couple of older updates at random: 20210724 works, 20210911 doesn't.

**Comment by dukeplay** [comment]
I have the same problem. The coverarts are shown in the browser, the coverarts are not shown on the local DSI display. In describing the problem, R101 can only join me, same hardware, same version.

I correct myself, the same problem also occurs with the browser on the PC. Coverart only appears after switching between playlist and playpage.

**Comment by R101G** [comment]
Fixed by 20211015 update - thanks.

Also, it's nice to see the 64 bit image has become the default for the Pi 3 and 4 :)
